tests/test_bit_kernels.py

```diff
@@ -0,0 +1,151 @@
+"""
+Test suite for bit vector kernels.
+Author: Ryan-Rhys Griffiths 2021
+"""
+
+import pytest
+import gpflow
+from gpflow.utilities import positive
+from gpflow.utilities.ops import broadcasting_elementwise
+import tensorflow as tf
+import torch
+from gpytorch.kernels import ScaleKernel
+from gprotorch.kernels.bit_vector_kernel import BitDistance, TanimotoKernel
+
+
+@pytest.mark.parametrize("x1, x2", [
+    (torch.ones((2, 2)), torch.ones((2, 2))),
+    (2*torch.ones((2, 2)), 2*torch.ones((2, 2))),
+    (10*torch.ones((2, 2)), 10*torch.ones((2, 2))),
+    (10000*torch.ones((2, 2)), 10000*torch.ones((2, 2))),
+    (0.1*torch.ones((2, 2)), 0.1*torch.ones((2, 2)))
+])
+def test_tanimoto_similarity_with_equal_inputs(x1, x2):
```

Comment:
Does this test need two args if you're only calling it when both args are the same (and the test would fail if they were different)?

Reply:
True, I've just left it as x1 and x2 as hopefully it'll be clearer what's going on to anyone who might look at the tests!